bokun.yaml

```diff
@@ -5688,56 +5692,73 @@ definitions:
       rateId:
         format: int64
         type: integer
+        description: 'Specifies the rate ID.'
       title:
         type: string
+        description: 'Specifies the title of the rate.'
       price:
         $ref: '#/definitions/PriceDto'
+        description: 'Specifies the rate price if it is set per booking. Field omitted if the price is set per person.'
       passengers:
         items:
           $ref: '#/definitions/CategoryPriceDto'
         type: array
+        description: 'The list of pricing categories and related prices per person. This field is omitted if the price is set per booking.'
       pickupPrice:
         $ref: '#/definitions/PriceDto'
+        description: 'Specifies the price for the pick up service on this rate.'
       dropoffPrice:
         $ref: '#/definitions/PriceDto'
+        description: 'Specifies the price for the drop-off service on this rate.'
       extras:
         items:
           $ref: '#/definitions/ExtraPriceDto'
         type: array
+        description: 'The list of prices for the extras available for this rate.'
   ActivityPriceListDateRange:
     type: object
     properties:
       from:
         type: string
+        description: 'Start date for the validity of this pricelist.'
       to:
         type: string
+        description: 'End date for the validity of this pricelist. Field omitted if the end date is not filled.'
       rates:
         items:
           $ref: '#/definitions/ActivityRatePricesDto'
         type: array
+        description: 'The list of rates that are set for each product on this pricelist.'
   PriceDto:
     type: object
     properties:
       currency:
         type: string
+        description: 'Specifies the currency on which the price is shown.'
       amount:
         format: double
         type: number
+        description: 'Specifies the amount of the price.'
       ofWhichTax:
         format: double
         type: number
+        description: 'Specifies the fraction of the amount that is dedicated for the taxes. Expressed in absolute value, applying the same currency as for amount.'
```

Comment:
Sorry, I don't understand this explanation :P

Reply:
Simplified :)